pkg/argocd/update.go

```diff
@@ -621,10 +621,15 @@ func getWriteBackConfig(app *v1alpha1.Application, kubeClient *kube.KubernetesCl
 	return wbc, nil
 }
 
-func parseDefaultTarget(appName string, path string) string {
-	defaultTargetFile := fmt.Sprintf(common.DefaultTargetFilePattern, appName)
+func parseDefaultTarget(appNamespace string, appName string, path string) string {
 
-	return filepath.Join(path, defaultTargetFile)
+	if (appNamespace == "argocd") || (appNamespace == "") {
```

Comment:
`argocd` should not be hard coded here, because Argo CD is regularly installed into different namespaces.

Reply:
So we should add a parameter to parseDefaultTarget so we can compare with the actual namespace where ArgoCD is deployed?

Reply:
We need to use the namespace as specified here: https://github.com/argoproj-labs/argocd-image-updater/blob/ae3eec3ff69487d13da770abd9167c61ac957931/cmd/run.go#L237

Reply:
Done